samples/matter/lock/README.rst

```diff
@@ -143,6 +143,26 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Setting Thread radio output power
```

Comment:
Ideally, I would like to have documentation about how to change the TX Power for all the transports (BLE, Thread, and Wi-Fi). Intuitively, it would be great to have a dedicated short page instead in the "Getting started with Matter" section in the documentation. Then.. we could link to this section from the example's doc. Thoughts?

Btw. in the future we should consider keeping 20dBm with FEM. 0 dBm for (S)SEDs and MAX TX Power (+8 - nRF52840 and +3 nRF5340) for non-sleepy end devices.

Reply:
So, for now, I will set the values you provided for all examples and remove that section from the samples. @greg-fer, what do you think, can we create a dedicated short page with instructions for setting TX power for all transports somewhere?

Reply:
Thanks Arek. Last comment.. would it make sense to set those defaults (3, 8, 20) in the sdk-connectedhomeip, and only change the TX power configuration for (S)SED devices?  In release notes we can then just mention that TX power has been changed to maximum output power for Light Bulb and Template?

Reply:
Ok, it make sense :)

Reply:
@ArekBalysNordic , how about here https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_radio_fem.html or -- depending how this TX is set -- here if it is set with a Kconfig https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_matter_gs_advanced_kconfigs.html or here if it is set with its own procedure https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_matter_intro_gs.html ?

Reply:
Regarding the first link - this is instruction for setting the radio in low-level implementation so we need something else, we can put it to the second link as it's Kconfig, but I think that the best option is to place it to the 3rd link. I think it is a good idea to create there a separate page for setting power for Thread, WiFi, and BLE. @LuDuda what do you think?